src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -27,7 +27,14 @@ def get_augmented_data(images, annotations, output_height, output_width, resize_
     # Suppresses verbosity to only errors
     tf.logging.set_verbosity(tf.logging.ERROR)
 
-    session = tf.Session()
+    graph = tf.Graph()
+    with graph.as_default:
+        with tf.Session() as session:
```

Comment:
I just saw this. Wouldn't you have to mention `with tf.Session(graph=graph) as session:`

Reply:
I believe the `graph.as_default()` takes care of that: any attempt to use the default graph, including by the `tf.Session` initializer, will use our local `graph` instance instead

Reply:
I think it should be `graph.as_default()`

Reply:
Yup, I got it working like that in some messed up code and then thought I could clean it up without testing before I pushed, sigh